<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitWidget</class>
 <widget class="QWidget" name="FitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>953</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="indexBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="prefix">
            <string>index: </string>
           </property>
           <property name="minimum">
            <number>-1</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="popbufButton">
           <property name="text">
            <string>Pop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearbufButton">
           <property name="text">
            <string>Clear Buffer</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTableWidget" name="bufferTable"/>
         <widget class="QTextEdit" name="resultEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="fitButton">
           <property name="text">
            <string>Fit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearfitButton">
           <property name="text">
            <string>Clear Fit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="methodBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="applyloadBox">
           <property name="text">
            <string>apply loaded</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="applyresultBox">
           <property name="text">
            <string>apply result</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="ParamTable" name="paramTable"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParamTable</class>
   <extends>QTableWidget</extends>
   <header>mahos.gui.param</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>loadButton</tabstop>
  <tabstop>bufferTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
